WireAuthentication/Sources/WireAuthenticationUI/Views/Login/NoHistory/NoHistoryView.swift

```diff
@@ -23,10 +23,39 @@ struct NoHistoryView: View {
     @ObservedObject var viewModel: NoHistoryViewModel
 
     var body: some View {
-        Color.red
+        VStack(spacing: 20) {
+            Text(L10n.Authentication.NoHistory.title)
+                .multilineTextAlignment(.center)
+                .font(.textStyle(.h2))
+                .lineLimit(nil)
+                .fixedSize(horizontal: false, vertical: true)
+            Text(L10n.Authentication.NoHistory.message)
+                .multilineTextAlignment(.center)
+                .wireTextStyle(.body1)
+                .lineLimit(nil)
+                .fixedSize(horizontal: false, vertical: true)
+            Button(action: {
+                viewModel.confirm()
+            }, label: {
+                Text(L10n.Authentication.NoHistory.confirm)
+            })
+            .wireButtonStyle(.primary)
```

Comment:
```suggestion
            Button(L10n.Authentication.NoHistory.confirm, action: viewModel.confirm)
                .wireButtonStyle(.primary)
                .bold()
```
1. The code can be written in a more compact way.
2. I haven't seen the designs, but I assume the button text should be bold.